rllib/BUILD

```diff
@@ -61,6 +61,29 @@
 # these tests in n different jobs.
 
 load("//bazel:python.bzl", "py_test_module_list")
+load("//bazel:python.bzl", "doctest")
+
+# TODO(@bveeramani): Enable doctests.
```

Comment:
RLLib has many failing doctests. Since it'll take a while to address those, I'd like to enable doctests for RLLib in a follow-up PR.